app/workers/asset_manager_create_whitehall_asset_worker.rb

```diff
@@ -1,7 +1,9 @@
 class AssetManagerCreateWhitehallAssetWorker < WorkerBase
   def perform(file_path, legacy_url_path)
     file = File.open(file_path)
-    Services.asset_manager.create_whitehall_asset(file: file, legacy_url_path: legacy_url_path)
+    asset_options = {file: file, legacy_url_path: legacy_url_path}
+    asset_options[:draft] = true if legacy_url_path =~ /attachment_data/
```

Comment:
Minor: I'm not keen on trailing `if` statements - I think the code is clearer if you use the block form.

Reply:
I generally agree if it's a long line but I'm happy that this is short enough not to be too unreadable so I'm going to leave it as is.